src/redux/actions/samples/loadSamples.js

```diff
@@ -1,58 +1,49 @@
 import fetchAPI from 'utils/http/fetchAPI';
 import handleError from 'utils/http/handleError';
-import fileNameForApiV1 from 'utils/upload/fileNameForApiV1';
 
 import {
   SAMPLES_LOADED,
   SAMPLES_ERROR,
   SAMPLES_LOADING,
 } from 'redux/actionTypes/samples';
 
-const toApiV1 = (samples, experimentId) => {
-  const apiV1Samples = {};
+const adaptedToRedux = (samples, experimentId) => {
```

Comment:
Can we just get rid of this or will it impact old experiments? Migration?

Reply:
what we receive from the api still doesn't completely match what is in redux, so I decided to keep this going for now